app/main/views/users.py

```diff
@@ -260,13 +260,13 @@ def export_users_for_framework(framework_slug):
                 application_result = 'no result'
             else:
                 application_result = 'pass' if sf.on_framework else 'fail'
-            framework_agreement = bool(sf.serialize()['agreementReturnedAt'])
+            framework_agreement = bool(getattr(sf.current_framework_agreement, 'signed_agreement_returned_at', None))
```

Comment:
You could use the `status` property of a framework agreement instead of the `signed_agreement_returned_at` if we want our code to be more reliant on the status field. (This way is also fine though)